resourcemonitor/resourcemonitor.go

```diff
@@ -74,13 +74,18 @@ type TrafficMonitoring interface {
 	GetInstanceTraffic(instanceID string) (inputTraffic, outputTraffic uint64, err error)
 }
 
+// PartitionParam partition information.
+type PartitionParam struct {
+	Name string `json:"name"`
+	Path string `json:"path"`
+}
+
 // Config configuration for resource monitoring.
 type Config struct {
-	aostypes.ServiceAlertRules
-	SendPeriod aostypes.Duration `json:"sendPeriod"`
-	PollPeriod aostypes.Duration `json:"pollPeriod"`
-	WorkingDir string            `json:"workingDir"`
-	StorageDir string            `json:"storageDir"`
+	aostypes.AlertRules
+	SendPeriod      aostypes.Duration `json:"sendPeriod"`
+	PollPeriod      aostypes.Duration `json:"pollPeriod"`
+	PartitionParams []PartitionParam  `json:"partitionParams"`
```

Comment:
Name it `partitions` or `partitionsInfo`

Reply:
Done

Reply:
Done